generator/build/search/createIndex.js

```diff
@@ -27,18 +31,23 @@ const getHtmlFiles = async (dir) =>
     const htmlFiles = await getHtmlFiles(htmlFilesDir);
     for (const key in htmlFiles) {
         const htmlContent = fs.readFileSync(`${htmlFilesDir}/${htmlFiles[key]}`).toString();
+
         const htmlContentMatch = htmlContent.match(/<div class="article">([\s\S]*?)<div class="footer-top">/gm);
         const titleMatch = htmlContent.match(/<h1 id="top">([\s\S]*?)<\/h1>/gm);
+        const headersMatch = htmlContent.match(/<h[^>]+>(.*)<\/h[^>]+>/gm);
```

Comment:
@aleksandrychev Should the regex require `<h` and then a number? We don't want to match other things like `<head>`, right?

Reply:
good point, changed regex to catch only h[1-6] elements